README.rst

```diff
@@ -23,6 +23,17 @@ Please don't commit changes with reStructuredText syntax errors that cause PEP
 generation to fail, or result in major rendering defects relative to what you
 intend.
 
+ReadTheDocs preview
+-------------------
```

Comment:
```suggestion

Browse the ReadTheDocs preview
------------------------------
```

* The other L3 headings are framed in imperative, so reworded this one to match
* (Minor nit) Added missing blank line before heading